### ISSUE TITLE
[FEATURE] Daily Affirmations and Inspirational Quotes

### ISSUE THREAD

**Issue by yashmandi** [algovengers/MindMate#98]
### Description
I believe that incorporating daily affirmations, motivational quotes, and mindfulness exercises can greatly enhance the user experience and contribute to a more positive and supportive environment

### Feature Details
1. As soon as the user logs in, the user should see the positive affirmation, should be on self-love, confidence and resiliance. 
2. Can create a different section which includes quotes and affirmations.

Would like to work on this feature under GSSoC '24. Request you to assign this task to me :)

**Comment by subharthihazra** [comment]
will it be a section on the homepage?

**Comment by yashmandi** [comment]
Yes, we can put somewhere below the 'Login' Button, similar to the toast.
![home-page](https://github.com/algovengers/MindMate/assets/132436131/1320d446-b983-4c9c-a00a-4e0de544daa8)
